client/lib/analytics/ad-tracking/load-tracking-scripts.js

```diff
@@ -15,11 +15,11 @@ import {
 	OUTBRAIN_SCRIPT_URL,
 	PINTEREST_SCRIPT_URL,
 } from './constants';
-
-// Ensure setup has run.
-import './setup';
+import { setupAllTrackingScripts } from './setup';
 
 export const loadTrackingScripts = attemptLoad( async () => {
+	setupAllTrackingScripts();
```

Comment:
Do we really need to call this? Seems like it's already called when at the time we `import` the module above.